evm/deploy-contract.js

```diff
@@ -137,6 +138,38 @@ async function getConstructorArgs(contractName, chain, wallet) {
             return [signers, threshold];
         }
 
+        case 'InterchainMultisig': {
+            const chainName = chain.axelarId;
+
+            const signers = contractConfig.signers;
+
+            if (!isAddressArray(signers)) {
+                throw new Error(`Missing InterchainMultisig.signers in the chain info.`);
+            }
+
+            const weights = contractConfig.weights;
+
+            if (!isNumberArray(weights)) {
+                throw new Error(`Missing InterchainMultisig.weights in the chain info.`);
+            }
+
+            const threshold = contractConfig.threshold;
+
+            if (!isNumber(threshold)) {
+                throw new Error(`Missing InterchainMultisig.threshold in the chain info.`);
+            }
```

Comment:
If you want you can use this function instead of a separate check for each: 
https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/evm/utils.js#L403

Reply:
Very neat!